src/components/RegistrationForm/Form/RegisterTerms/RegisterTerms.jsx

```diff
@@ -0,0 +1,19 @@
+import { StyledText } from './RegisterTerms.styled.js';
+import { NavLink } from 'react-router-dom';
+
+const RegisterTerms = () => {
+  return (
+    <StyledText>
+      Реєструючись, ви погоджуєтесь з умовами <br />
+      <NavLink to={'/'}>
+        <span className="text"> положення про обробку і захист персональних даних </span>
```

Comment:
why do you use another "span" inside if you can give class to NavLink instead, and style it or directly style to tag "a".

Reply:
fixed